backend/info_manager_dict.py

```diff
@@ -0,0 +1,31 @@
+import dataclasses
+
+@dataclasses.dataclass
+class Place:
+    name: str
+    ID: str
+    latitude: float
+    longitude: float
+
+@dataclasses.dataclass
+class UserInfo:
+    home: tuple[float, float]
+    places: list[Place]
+
+class InfoManagerDict:
+
+    def __init__(self):
+        self.data = dict()
+
+    def add_user(self, latitude, longitude):
+        user_id = str(len(self.data))
```

Comment:
I think we should have `user_id` be set to a uuid that we create here in the backend. What do you think?